server/src/controllers/Events/resolver.ts

```diff
@@ -134,38 +156,66 @@ const updateReminders = (event: EventWithUsers, startAt: Date) => {
   }
 };
 
-const hasVenueChanged = (data: EventInputs, event: EventWithUsers) =>
+const hasVenueLocationChanged = (data: EventInputs, event: EventWithUsers) =>
   data.venue_type !== event.venue_type ||
-  (isOnline(event.venue_type) && data.streaming_url !== event.streaming_url) ||
   (isPhysical(event.venue_type) && data.venue_id !== event.venue_id);
+const hasDateChanged = (data: EventInputs, event: EventWithUsers) =>
+  !isEqual(data.ends_at, event.ends_at) ||
+  !isEqual(data.start_at, event.start_at);
+const hasStreamingUrlChanged = (data: EventInputs, event: EventWithUsers) =>
+  isOnline(event.venue_type) && data.streaming_url !== event.streaming_url;
 
 const buildEmailForUpdatedEvent = async (
   data: EventInputs,
   event: EventWithUsers,
 ) => {
-  const subject = `Venue changed for event ${event.name}`;
-  let venueDetails = '';
+  const subject = `Details changed for event ${event.name}`;
 
-  if (isPhysical(event.venue_type)) {
+  // this returns [object Promise] and I don't know why
+  const createVenueLocationContent = async () => {
     const venue = await prisma.venues.findUniqueOrThrow({
       where: { id: data.venue_id },
     });
+
     // TODO: include a link back to the venue page
-    venueDetails += `The event is now being held at <br>
-${venue.name} <br>
-${venue.street_address ? venue.street_address + '<br>' : ''}
-${venue.city} <br>
-${venue.region} <br>
-${venue.postal_code} <br>
+    return `The event is now being held at <br />
+    <br />
+- ${venue.name} <br />
+- ${venue.street_address && venue.street_address + '<br />'}
+- ${venue.city} <br />
```

Comment:
```suggestion
- ${venue.street_address ? venue.street_address + '<br />- ' : ''}
${venue.city} <br />
```
Otherwise you can end up with messages like

> \- Venue
> \- null - 123
> \- 123